paramnb/__init__.py

```diff
@@ -260,7 +260,7 @@ def path_change_event(event):
                 selector.options=named_objs(p_obj.get_range().items())
 
                 if p_obj.objects and not self.p.button:
-                    self.execute(None)
```

Comment:
I *think* the change is correct, but I can't seem to track down the original purpose of this line.  Since it looks correct and appears to fix a problem, I guess we should just merge it.